IGI.Lab3/CarPark.WebUI/Models/AccidentViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using CarPark.BLL.Models;
+
+namespace CarPark.WebUI.Models
```

Comment:
I guess folder should have name ViewModels

Reply:
fix